src/containers/shared/components/Transaction/NFTokenCreateOffer/parser.ts

```diff
@@ -1,6 +1,12 @@
-const formatAmount = require('./formatAmount')
+import { NFTokenCreateOfferInstructions } from './types'
+import { TransactionParser } from '../types'
 
-module.exports = (tx, meta) => {
+const formatAmount = require('../../../../../rippled/lib/txSummary/formatAmount')
```

Comment:
use import instead of require.

Reply:
When should require be used over import? I saw you use require for requiring utils for one of parsers in SignerListSet PR I think so thought this was similar

Reply:
That must have been an oversight on my part

Reply:
It's really a JS vs TS thing

Reply:
Oh i did it to avoid a circular dependency.  Will break out the function i was calling to its own file

Reply:
I put in a PR to remove require from SignerListSet #326